misk/src/main/kotlin/misk/web/dashboard/AdminDashboardModule.kt

```diff
@@ -33,6 +36,9 @@ class AdminDashboardModule(private val isDevelopment: Boolean) : KAbstractModule
     // Install base dashboard support
     install(DashboardModule())
 
+    bind<AdminDashboardProtobufDocUrlPrefix>()
```

Comment:
Thinking about it some more... I think this is fine, let's leave it as a constructor parameter for now.

Reply:
Acutally, having second thoughts here, let's leave it as a constructor parameter and if it comes up in practice that we don't want it then we can swap it then.

Reply:
Yeah, I was thinking if we allow this to be set in the constructor then skim could set it here: https://github.com/squareup/cash-skim/blob/fa9a874b0f777757d2a71da06bafdb02ba186b22/skim/src/main/kotlin/com/squareup/skim/SkimRealModule.kt#L71

Reply:
The other approach which I'm now leaning towards again is making the binding optional and Skim can do the binding itself rather than the binding in this module. That's the more Guice-y way to do it, avoiding constructor parameters.

Reply:
Example of an optional binding here: https://github.com/cashapp/misk/blob/9593078083385e235dda816af539bd48bae616c3/misk/src/main/kotlin/misk/web/metadata/ServiceMetadataAction.kt#L34-L42